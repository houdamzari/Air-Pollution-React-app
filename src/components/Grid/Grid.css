.grid {
  display: grid;
  grid-auto-columns: 1fr;
  grid-template-columns: auto auto;
  grid-template-rows: auto auto auto;
  gap: 20px 20px;

  width: 100%;
}
