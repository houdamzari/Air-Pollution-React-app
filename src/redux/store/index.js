import { configureStore } from "@reduxjs/toolkit";
import citiesReducer from "../slices/citiesSlice";
import polutionReducer from "../slices/airPolutionSlice";
export const store = configureStore({
  reducer: {
    cities: citiesReducer,
    airpolution: polutionReducer,
  },
});
