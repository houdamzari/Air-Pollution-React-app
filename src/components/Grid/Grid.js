import "./Grid.css";
function Grid({ children }) {
  return <div className="grid">{children}</div>;
}

export default Grid;
