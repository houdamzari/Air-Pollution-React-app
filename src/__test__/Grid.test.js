import "@testing-library/jest-dom";
import { Provider } from "react-redux";
import store from "../redux/store/index";
import Grid from "../components/Grid/Grid";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";

describe("Test Grid Component", () => {
  test("Grid component should match with snapshot", () => {
    const grid = renderer
      .create(
        <BrowserRouter>
          <Grid />
        </BrowserRouter>
      )
      .toJSON();

    expect(grid).toMatchSnapshot();
  });

});
