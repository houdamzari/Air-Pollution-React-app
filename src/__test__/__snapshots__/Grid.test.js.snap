// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Grid Component Grid component should match with snapshot 1`] = `
<div
  className="grid"
/>
`;
